### PR TITLE
Profile form

### PR DESCRIPTION
**Pull request by uzmahamid01**
profile page with signed in user info
and degree planner page with basic form and default degree plan provided by tamu

### CODE REVIEW

File: app/controllers/def_degree_controller.rb
Comment (nealetham) [inline app/controllers/def_degree_controller.rb:1]: What is this file for?

**Review by nealetham — DISMISSED** [review]
Other than that comment, and fixing the merge conflict, this should be good to go!

File: app/controllers/def_degree_controller.rb
Comment (uzmahamid01) [reply app/controllers/def_degree_controller.rb:1]: there were few redirection issues they are fixed now.

**Review by mariaeviteri — APPROVED** [review]
Looks good, nice job.

**Review by uzmahamid01 — COMMENTED** [review]
Redirection is fixed

**Pull request merged**